webapp/Gruntfile.js

```diff
@@ -30,7 +30,25 @@ module.exports = function(grunt) {
       TerraMA2WebAppMin: {
         options: {
           baseUrl: SCRIPTS_PATH,
-          out: DEST_PATH + "terrama2-webapp.min.js",
+          optimize: 'none',
```

Comment:
should this line be here?

Reply:
Yes, it is.

The native grunt plugin for minification does not support ES5+ expressions. In this way, we should disable the process and keep optimization as `none` and then perform customization on attribute `out`. Take a look in that property. We need to retrieve a function and write manually both minified file and `.map` file using module `uglify-es`.